Islandora-Devops/islandora-starter-site
# Add Mirador display for Paged Content.

# What does this pull request do?

1. Adds the islandora_mirador module as a dependency.
2. Adds context reaction to show Mirador for Paged Content
3. Disables the Openseadragon Paged Content context reaction.

# How should this be tested?


Run composer install.

Import the site's configuration using Drush:
```bash
drush config:import 
```



Add a new Repository Item content with model Paged Content.

Click on the Children tab and choose 'Batch upload children"

Choose Repository Item, and Page for content model, and File for media type.

On the next page, upload one or more TIFF files.

Go back to the View tab ob the object.

You should see the Mirador viewer with the page content you uploaded.

For single page items Openseadragon is still the default.
